parsing/src/parts.ts

```diff
@@ -227,7 +252,7 @@ function isCollectable(ingredients: string): boolean {
     return collectableDescriptors.some(descriptor => ingredients.includes(descriptor));
 }
 
-function stackSizeConvert(stackSize: string) {
+function stackSizeConvert(stackSize: string):number {
```

Comment:
```suggestion
function stackSizeConvert(stackSize: string) :number {
```
Lint bonk

Reply:
What is wrong with mine? the space before the :?